server/src/main/java/org/spine3/server/aggregate/AggregateStorage.java

```diff
@@ -56,6 +56,14 @@ protected AggregateStorage(boolean multitenant) {
         super(multitenant);
     }
 
+    /**
+     * Reads a {@link AggregateStateRecord} by the passed ID.
+     *
+     * @param aggregateId the ID of the record to read
```

Comment:
This description is not formatted.

I also suspect it has been copied from its parent, so you might use `{@inheritDoc}` instead.